media/css/mozorg/contribute/contribute-2020.scss

```diff
@@ -0,0 +1,387 @@
+    }
+
+    .contribute-intro-media-wrapper {
+        @include bidi(((left, -10px, auto),(right, auto, -10px),));
+        position: absolute;
+        width: 100%;
+
+
+        @media #{$mq-lg} {
+            @include bidi(((right, -25px, auto),(left, auto, -25px),));
+            left: auto;
+            top: 0;
+            width: 50%;
+        }
+    }
+}
+.contribute-section-subheading {
+    @include text-title-xs;
+}
+
+.contribute-hero {
+    background-color: $color-marketing-gray-20;
+    margin-bottom: $layout-md;
+    padding-bottom: $layout-sm;
+    padding-top: $layout-sm;
+
+    @media #{$mq-lg} {
+        margin-bottom: $layout-lg;
+    }
+}
+
+.contribute-card-container {
+    @media #{$mq-sm} {
+        display: flex;
+        flex-wrap: wrap;
+    }
+
+    .contribute-card {
+        display: flex;
+        flex: 1 0 45%;
+
+        @media #{$mq-md} {
+            flex: auto;
+        }
+    }
+
+    .contribute-card-wrapper {
+        background: $color-white;
+        display: flex;
+        flex-direction: column;
+        transition: transform 150ms ease-in-out,
+            -webkit-transform 150ms ease-in-out;
+
+        &:hover,
+        &:focus {
+            @include transform(translateZ(0) scale(1.02));
+            backface-visibility: hidden;
+        }
+    }
+
+    .contribute-card-content {
+        flex: 1;
+        padding: $spacing-md;
+    }
+
+    .contribute-card-content-tags {
+        margin-bottom: $spacing-sm;
+        min-height: $layout-md;
+    }
+
+    .contribute-card-tag-bold {
+        font-weight: bold;
+    }
+}
+
+.contribute-banner-wrapper {
+    @include border-box;
+    margin: 0 auto;
+    max-width: $content-max;
+    min-width: $content-xs;
+
+    @media #{$mq-md} {
+        padding: 0;
+    }
+
+    @media #{$mq-lg} {
+        margin-bottom: $layout-xl;
+        padding: $layout-xs $layout-xl 0;
+    }
+}
+
+.contribute-banner {
+    padding: $layout-xl $layout-md $layout-md;
+
+    @media #{$mq-lg} {
+        align-items: center;
+        display: flex;
+        justify-content: space-between;
+        min-height: 260px;
+        padding-bottom: 0;
+        padding-top: 0;
+        position: relative;
+    }
+
+    .contribute-banner-content {
+        @media #{$mq-lg} {
+            @include bidi(((margin-left, 200px, auto),(margin-right, auto, 200px),));
+            width: 60%;
+            max-width: 495px;
+        }
+    }
+
+    .contribute-banner-cta {
+        flex: 1 0 auto;
+
+        @media #{$mq-lg} {
+            @include bidi(((margin-left, $layout-xl, auto),(margin-right, auto, $layout-xl),));
+        }
+    }
+}
+
+.contribute-banner-gethelp {
+    background: $color-ink-80 url('/media/img/contribute/contribute-gethelp-banner-bg.jpg') center top no-repeat;
+    background-size: cover;
+    color: $color-white;
+
+    @media #{$mq-lg} {
+        margin-bottom: $layout-lg;
+    }
+
+    .contribute-banner-gethelp-title {
+        @include font-size(24px);
```

Comment:
I'm assuming these custom font-sizes have already been discussed and we're Ok with this page setting arbitrary values?